lib/shipit/command.rb

```diff
@@ -215,5 +214,40 @@ def parse_arguments(arguments)
       end
       return args, options
     end
+
+    def running?
+      !!pid && !@status
+    end
+
+    def code
+      @status&.exitstatus
+    end
+
+    def signaled?
+      @status.signaled?
```

Comment:
Do we need an & here like above?

Reply:
As this is a new API I decided to leave it unguarded, to catch anything in future calling this while the process is alive. No strong feelings if you prefer defensive here.

Reply:
No, let's not introduce unnecessary defensiveness if it's not needed.